Library/Homebrew/test/utils/github_spec.rb

```diff
@@ -57,10 +57,10 @@
     end
   end
 
-  describe "::sponsors_by_tier", :needs_network do
+  describe "::sponsorships", :needs_network do
     it "errors on an unauthenticated token" do
       expect {
-        described_class.sponsors_by_tier("Homebrew")
+        described_class.sponsorships("Homebrew")
       }.to raise_error(/INSUFFICIENT_SCOPES|FORBIDDEN|token needs the 'admin:org' scope/)
```

Comment:
This, supposedly, should be changed to a lower scope, right?

Reply:
@Rylan12 Yep, but I think it just won't fail at all now 🤷🏻.